manifests/o/OpenVPNTechnologies/OpenVPN/2.5.037/OpenVPNTechnologies.OpenVPN.installer.yaml

```
@@ -0,0 +1,36 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OpenVPNTechnologies.OpenVPN
+PackageVersion: 2.5.037
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install

```

```suggestion
UpgradeBehavior: install
ReleaseDate: 2022-11-02
```
Nit: More readable with release date not at the bottom